serbian/start.md

```diff
@@ -12,3 +12,5 @@ Mlada radnica ga je ljubazno gledala. Rekla je:
 "Ili mozda tri?"[klikni ovde](noveIdeje/noveIdeje.md)
 
 "Da li ste mozda zainteresovani za neki dezert?" [klikni ovde](dezert/dezert.md)
+ 
+"Ustvari nije bio mamuran, nije cak ni bio tu u pekari, to je sve bilo u njegovoj glavi..." [klikni ovde](mozda-nesto-drugacije/zalepio-slicicu.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.